controller/WebController.php

```diff
@@ -178,7 +178,7 @@ public function invokeVocabularyConcept(Request $request)
         if ($this->notModified($results[0])) {
             return;
         }
-        $pluginParameters = $vocab->getConfig()->getPluginParameters();
+        $pluginParameters = json_encode($vocab->getConfig()->getPluginParameters(), true);
```

Comment:
What is the purpose on `true` parameter used here and elsewhere? See https://www.php.net/manual/en/function.json-encode.php